app/src/androidTestWebview/java/org/mozilla/focus/activity/ImageSelectTest.java

```diff
@@ -56,6 +79,18 @@ protected void beforeActivityLaunched() {
                     .putBoolean(FIRSTRUN_PREF, true)
                     .apply();
 
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                rabbitImage = TestHelper.mDevice.findObject(new UiSelector()
+                        .resourceId("rabbitImage")
+                        .enabled(true));
+            }
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 13 should be on the same line as the next part of a multi\-block statement \(one that directly contains multiple blocks: if/else\-if/else, do/while or try/catch/finally\)\.